app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()[1:]
```

Comment:
I'd think you should introduce variables for each argument so that code is easier to read.